### ISSUE TITLE
Add example for excluding all methods in an assembly

### ISSUE THREAD

**Issue by maxcalvin** [MicrosoftDocs/visualstudio-docs#3865]
It's useful to be able to exclude an entire assembly (for example, acceptance tests) that should not be included in live unit testing. I don't know of the ways to do it in the other test systems but with xUnit you can use:

[assembly: AssemblyTrait("Category", "SkipWhenLiveUnitTesting")]

It took way too much googling on bing to find this solution, so it would be nice to have it included here since this is where people start to look.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3113c2af-d97e-7b35-17ce-c675c3dff889
* Version Independent ID: b716c4af-d92b-cb3b-8af0-1b3ada479ac0
* Content: [Live Unit Testing - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing?view=vs-2019#include-and-exclude-test-projects-and-test-methods)
* Content Source: [docs/test/live-unit-testing.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/test/live-unit-testing.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-test**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

**Comment by WilliamAntonRohm** [comment]
@maxcalvin -- Max, thank you for your feedback and providing an example.

@rpetrusha -- Ron, please look into his example.

**Comment by maxcalvin** [comment]
Thanks William, after raising the issue I went and had a glance at the other test frameworks in the example, and thanks to their code being open source on Github I could see that their attributes were tagged as applicable to assemblies. 

So I raised a PR with an update adding examples for assembly blocking based on those finds, but I've only checked the xUnit version. No worries if the PR is rejected just trying to contribute a little.

Cheers!
